ui/lib/core/addon/templates/components/replication-action-demote.hbs

```diff
@@ -1,9 +1,9 @@
 <div class="action-block is-rounded" data-test-demote-replication>
   <div class="action-block-info">
-    <h4 class="title is-5 is-marginless">
+    <h3 class="title is-5 is-marginless">
       Demote cluster
-    </h4>
-    <p>
+    </h3>
+    <p class="has-top-padding-s">
```

Comment:
There was some ugliness I addressed. Not a result of my h4 -> h3 change.

Before 
<img width="360" alt="image" src="https://github.com/hashicorp/vault/assets/6618863/a6608e3e-8c17-40c3-97d4-ffff9dbf643a">

After
<img width="548" alt="image" src="https://github.com/hashicorp/vault/assets/6618863/7b92a486-fd3a-4b06-bab5-53080ccb9166">